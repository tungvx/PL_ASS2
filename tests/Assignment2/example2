A=test
B=more complex
%
this is a B A
