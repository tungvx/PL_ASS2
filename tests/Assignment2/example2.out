this is a more complex test
